src/pages/User/components/PullRequests/UserShare.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+export default class UserShare extends Component {
```

Comment:
Can we use the function method? To be consistent with other components in the app?

`const UserShare = () => (`